pkg/graph/schema.graphql

```diff
@@ -740,7 +740,14 @@ type Mutation {
   createAccessibilityRequestDocument(
     input: CreateAccessibilityRequestDocumentInput!
   ): CreateAccessibilityRequestDocumentPayload
-  createSystemIntakeActionNotItRequest(input: BasicActionInput): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionBusinessCaseNeeded(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionBusinessCaseNeedsChanges(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionGuideReceievedClose(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionNoGovernanceNeeded(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionNotItRequest(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionNotRespondingClose(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionReadyForGRT(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
+  createSystemIntakeActionSendEmail(input: BasicActionInput!): UpdateSystemIntakePayload @hasRole(role: EASI_GOVTEAM)
```

Comment:
A lot of things I've read have suggested that each mutation should have its own Input and Payload types. Since we are building an internal API and control the only client that advice is somewhat less relevant, but it's still something worth keeping in mind.

A middle ground would be to always have a 1:1 between inputs and Payloads. In this case we could have a `BasicActionPayload` for all of these basic action mutations instead of reusing `UpdateSystemIntakePayload`, which is really unrelated to these mutations.

Reply:
Another reason to do this that I just thought of is that it might make sense to return the action that was created. I don't think there is an easy way to do that without fetching the entire collection of actions using `UpdateSystemIntakePayload`. I envision us doing something like:

```graphql
type BasicActionPayload {
  systemIntake: SystemIntake
  action: SystemIntakeAction (or whatever)
  userErrors: [UserError!]
}
```

Since we don't have a representation of actions in the graph yet, this could be something that we do once those are in place. So perhaps we do a refactoring once all the functionality is in place?